site/content/entry/yuzu-mini-dev-2/index.md

```diff
@@ -0,0 +1,133 @@
+
+**R:** I've worked on the graphics department and the compute department of Switch emulation.
+For those that don't know, the Switch's GPU - it's a Nvidia Tegra X1, the same architecture of a GTX 960+.
+Emulating a modern GPU (Tegra) by using other modern GPUs (user end), is my area of interest. 
+
+As for why I like it, it's because GPUs nowadays are like mini sub-computers inside a machine.
+The list of functionality present in the Tegra X1 grows bigger the more we look into it; since it’s a Nvidia device, it can execute CUDA kernels (programs that run some computations in the GPU, primarily used for scientific calculations or accelerated computations).
+It has the 5 traditional shader stages, an extra vertex shader stage, compute shaders, while at the same time it supports the supposedly deprecated features from the D3D9 era like rendering with quads and alpha testing.
+
+From time to time, I try to get into Core emulation, but it never ends up with good results. Funnily enough, while I was investigating the SSBU crash, I totally thought I made a breakthrough  - only to find out I was disassembling the wrong module. (LOL)
+
+
+**Q: What are some of the challenges you generally face, when working on GPU emulation?**
+
+**R:** One of biggest challenges, is the inherent variable state of Nvidia GPUs.
+Sometimes graphics APIs like OpenGL and Vulkan have requirements that are the "common ground", something that's needed by the lowest denominator.
+We have to write very annoying workarounds to get that functionality working in the emulator.
+
+<p style="color:cyan"><b><i>Here are some sneak peeks from my Vulkan test build. It's still missing a lot of things, but it's not too bad either.</b></i></p>
+
+{{< imgs
+   "./botw.png|The Legend of Zelda - Breath of the Wild"
+   "./onepiece.png|One Piece - Unlimited World"
+   "./smo.png|Super Mario Odyssey"
+>}}
+
+**Q: What motivates you every day to keep working on this stuff?**
+
+**R:** Emulating modern GPU features on hardware with those same features, is something that has only been possible in the recent times, due to the increasing similarities in hardware between consoles and desktops.
+Slowly understanding how some known functionality is implemented by the hardware, and then re-implementing it on the emulator afterwards, really motivates me.
+
+There are two things that I enjoy the most:
+- seeing games run faster and better than how they ran before and
+- seeing games get more and more closer to how they look on the console.
+
+And if I manage to get these improvements by writing better code, it's an added bonus.
+
+**Q: Can you tell us a little about what you are currently working on? A sneak-peek perhaps?**
+
+**R:** That's classified information :P
+I've recently worked on a improved & generic texture cache. 
+Blinkhawk helped by cleaning up my bugs to get the thing working and he's done an awesome job.
```

Comment:
I'd suggest linking to Blinkhawk's github profile like we did with Rodrigo. The first reference of another developer should include a link to their profile.